README.md

```diff
@@ -21,9 +21,9 @@ Below are the steps in the data flow diagram:
 
 - fetcher.py script, that represents the fetcher DAG, would retrieve the data from the current weather API.
 
-- The fetcher script would process and clean the data, then stores it the Postgres database considering relationships, integrity, performance, and extendability.
+- The fetcher script would process and clean the data, then stores it the Postgres database considering relationships, integrity, performance, and extendability. We make a basic version of it for you to save your time. You can improve it if you need to. For example, you will need to add 10 more cities to the list of cities for the next step.
```

Comment:
Since this is a public repo, will making the basic DB schema available essentially allow the `takehome-data-eng` folks to cheat?

`We make a basic version of it for you to save your time...` - should we make it clearer what `it` is?

Reply:
Well, we ask about the detail of the design anyways. 

I will fix it.